docs/streams.html

```diff
@@ -279,8 +279,8 @@
 <pre>
     KStreamBuilder builder = new KStreamBuilder();
 
-    KStream<String, GenericRecord> source1 = builder.stream("topic1", "topic2");
-    KTable<String, GenericRecord> source2 = builder.table("topic3", "stateStoreName");
+    KStream&lt;String, GenericRecord&gt; source1 = builder.stream("topic1", "topic2");
```

Comment:
Why is this required? It is within `<pre>` environment. Same applies to other changes similar to this one.